test/stress/mt_batching_test.go

```diff
@@ -0,0 +1,119 @@
+package test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/weaviate/weaviate-go-client/v4/test/testsuit"
+	"github.com/weaviate/weaviate-go-client/v4/weaviate/filters"
+	"github.com/weaviate/weaviate-go-client/v4/weaviate/graphql"
+	"github.com/weaviate/weaviate/entities/models"
+)
+
+func TestMTBatching_stress(t *testing.T) {
+	className := "Collection"
+	client := testsuit.CreateTestClient(true)
+	ctx := context.Background()
+
+	defer func() {
+		err := client.Schema().ClassDeleter().WithClassName(className).Do(ctx)
+		require.Nil(t, err)
+	}()
+
+	t.Run("Create multi-tenancy collection", func(t *testing.T) {
```

Comment:
can we have this outside of a t.Run() ?
Otherwise you will aways have to run the complete test and cannot run the individual subtest on their own